app/build.gradle

```diff
@@ -36,7 +36,7 @@ dependencies {
     // There is a better way to declare and structure dependencies.
     // However, that is a different topic.
     def daggerVersion = '2.13'
-    def butterKnifeVersion = '8.7.0'
+    def butterKnifeVersion = '8.8.0'
```

Comment:
This is the only code change required for this upgrade.

Reply:
The Butterknife Gradle plugin defined in the root build.gradle should have also been changed to `8.8.0`. 

> Fixed in https://github.com/vestrel00/android-dagger-butterknife-mvp/pull/71/files#r173835746